wurst/dummy/DummyRecycler.wurst

```diff
@@ -143,6 +145,10 @@ init
 			DummyRecycler.angleQueues[i].enqueue(createDummy(boundMax, DUMMY_PLAYER, (i* ANGLE_DEGREE).asAngleDegrees()))
 
 @compiletime function generateDummyUnit()
+	let a = new AbilityDefinitionPermanentInvisibility(ABIL_ID_GEN.next())
+	a..presetAutoAcquireAttackTargets(_ -> false)
```

Comment:
I think there's no need for a `let` binding here. Just `new X()` followed by cascades.

Reply:
I agree. There was a `let` binding in the previous version (on the creation of UnitDefinition), so I decided not to remove it since it was in stdlib already and to do the rest in the same style. Gonna refactor it now.